NDTensors/ext/NDTensorsAMDGPUExt/NDTensorsAMDGPUExt.jl

```diff
@@ -0,0 +1,12 @@
+module NDTensorsAMDGPUExt
+using Functors
```

Comment:
This should be removed from here as well.

Reply:
I think the only thing we are using from `Functors` is `fmap`.

Reply:
Sorry I missed this, I ran the unittests and forgot about looking into this. Thank you